### ISSUE TITLE
use pcall to include darkrp modules

### ISSUE THREAD

**Issue by ExtReMLapin** [FPtje/DarkRP#2746]
some modules are coded like shit and when one of them errors, it breaks the whole modules loading, use pcall pls

**Comment by roboderpy** [comment]
Errors weren't created to just "be present". They were created for being annoying and forcing monkeys to fix their code.

**Comment by ExtReMLapin** [comment]
yeah but retarded darkrp owners break shit and then complain to us

**Comment by roboderpy** [comment]
The more error gets buried by try catch, the harder it becomes to debug code when breakage happens.

**Comment by ExtReMLapin** [comment]
yeah true
